applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -540,6 +786,16 @@ void EmbeddedSkinVisualizationProcess::CreateVisualizationGeometries()
     r_comm.Barrier();
 }
 
+template<class TDataType>
+void EmbeddedSkinVisualizationProcess::InitializeNonHistoricalVariables(const std::vector<const Variable<TDataType>*>& rNonHistoricalVariablesVector)
+{
+    for (auto& r_node : mrVisualizationModelPart.Nodes()) {
```

Comment:
No OMP here?

Reply:
I did an `auto` loop. I'd change it to make it parallel.